unity-ggjj/Assets/Scripts/NarrativeScript/NarrativeScriptPlayer/NarrativeScriptPlayerComponent.cs

```diff
@@ -45,16 +45,20 @@ public void TryPressWitness()
     }
 
     /// <summary>
-    /// Loads a narrative script, ending the current narrative script
-    /// and continuing the beginning of the loaded script
+    /// Loads a narrative script using a given narrative script name,
+    /// ending the current narrative script and
+    /// continuing the beginning of the loaded script
     /// </summary>
     /// <param name="narrativeScriptName">The name of the narrative script to load</param>
     public void LoadScript(string narrativeScriptName)
     {
         var narrativeScriptText = Resources.Load<TextAsset>($"InkDialogueScripts/{narrativeScriptName}");
-        NarrativeScriptPlayer.ActiveNarrativeScript = new NarrativeScript(narrativeScriptText);
+        Debug.Assert(narrativeScriptName != null, $"Failed to load narrative script with name {narrativeScriptName}");
```

Comment:
I think this should be `narrativeScript>Text< != null ` rather than `narrativeScript>Name< != null`?  

Also, when displaying file-related error messages, I'm always a fan of printing the absolute path. With Unity we should probably also mention that it's a [Resource](https://docs.unity3d.com/ScriptReference/Resources.html) we're trying to load, that way a user knows where the file should be.
```suggestion
        Debug.Assert(narrativeScriptText != null, $"Failed to load narrative script resource at 'InkDialogueScripts/{narrativeScriptName}'");
```